CMakeLists.txt

```diff
@@ -0,0 +1,177 @@
+cmake_minimum_required(VERSION 3.1)
+cmake_policy(SET CMP0074 NEW)
```

Comment:
If you run this code with CMake 3.2 (minimum required) you will run into error "Policy ... is not known to this version of CMake." You should update the minimum required version or use `if(POLICY ...)`:
- https://cgold.readthedocs.io/en/latest/tutorials/version-policies.html#keep-using-old